pkg/apis/networking/v1beta1/BUILD

```diff
@@ -5,6 +5,7 @@ load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
 go_library(
     name = "go_default_library",
     srcs = [
+        "conversion.go",
         "defaults.go",
```

Comment:
Do we need corresponding changes to `defaults.go` and `conversion.go` files for v1? Not seeing them modified in this PR.

Reply:
v1 shouldn't require custom conversion, since the DefaultBackend field name matches in internal/external types